packages/yourloops/test/integration/pages/notification/notification-for-patient.spec.tsx

```diff
@@ -0,0 +1,78 @@
+ */
+
+import { act, within, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { renderPage } from '../../utils/render'
+import { mockPatientLogin } from '../../mock/patient-login.mock'
+import { monitoredPatientAsTeamMember } from '../../mock/patient.api.mock'
+import NotificationApi from '../../../../lib/notifications/notification.api'
+import { iTeamOne, teamOne, teamTwo } from '../../mock/team.api.mock'
+import TeamAPI from '../../../../lib/team/team.api'
+import { invitationTeam } from '../../data/notification.data'
+import { closeDialogNotificationTeam } from '../../assert/notification-join-team'
+
+describe('Notification page for patient', () => {
+  beforeAll(() => {
+    mockPatientLogin(monitoredPatientAsTeamMember)
+    jest.spyOn(TeamAPI, 'getTeams').mockResolvedValue([teamOne, teamTwo])
+    jest.spyOn(NotificationApi, 'getReceivedInvitations').mockResolvedValue([invitationTeam])
+    jest.spyOn(TeamAPI, 'getTeamFromCode').mockResolvedValue(iTeamOne)
+    jest.spyOn(NotificationApi, 'acceptInvitation').mockResolvedValue()
+  })
+
+  it('should be able to accept a team invite', async () => {
+    await act(async () => {
+      renderPage('/notifications')
+    })
+
+    const acceptButton = screen.getByRole('button', { name: 'Accept' })
+    await userEvent.click(acceptButton)
+    const dialogInputCodeTeam = screen.getByRole('dialog')
+    const inputCode = within(dialogInputCodeTeam).getByRole('textbox')
+    const addTeamButton = within(dialogInputCodeTeam).getByRole('button', { name: 'Continue' })
+    expect(addTeamButton).toBeDisabled()
+    await userEvent.type(inputCode, '263381988')
+    expect(addTeamButton).toBeEnabled()
+    await userEvent.click(addTeamButton)
+    const dialogPrivacy = screen.getByRole('dialog')
+    const addCareTeamButton = within(dialogPrivacy).getByRole('button', { name: 'Join team' })
+    const checkPolicy = within(dialogPrivacy).getByRole('checkbox')
+    expect(dialogPrivacy).toHaveTextContent('Share your data with a care teamYou ' +
+      'are about to share you data withiTeamOne6 rue des champs75000Paris679517388Please verify that the above details match the information provided by your healthcare professional ' +
+      'before accepting the invitation.By accepting this invitation, I recognize this team as my care team and consent to share my personal data with all its members, ' +
+      'who are authorized healthcare professionals registered on YourLoops. ' +
+      'I acknowledge that I can revoke this access at any time.Read our Privacy Policy for more information.CancelJoin team')
+    await userEvent.click(checkPolicy)
+    expect(addCareTeamButton).toBeEnabled()
+    await userEvent.click(addCareTeamButton)
+    expect(NotificationApi.acceptInvitation).toHaveBeenCalledWith('unmonitoredPatientId', invitationTeam)
+    expect(screen.queryByRole('dialog')).not.toBeInTheDocument()
+
+    closeDialogNotificationTeam()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏Missing `await` before function call